table.csv

```diff
@@ -1,462 +1,462 @@
+blake2b-224,                    multihash,      0xb21c,         permanent,
+blake2b-232,                    multihash,      0xb21d,         permanent,
+blake2b-240,                    multihash,      0xb21e,         permanent,
+blake2b-248,                    multihash,      0xb21f,         permanent,
+blake2b-256,                    multihash,      0xb220,         permanent,
+blake2b-264,                    multihash,      0xb221,         permanent,
+blake2b-272,                    multihash,      0xb222,         permanent,
+blake2b-280,                    multihash,      0xb223,         permanent,
+blake2b-288,                    multihash,      0xb224,         permanent,
+blake2b-296,                    multihash,      0xb225,         permanent,
+blake2b-304,                    multihash,      0xb226,         permanent,
+blake2b-312,                    multihash,      0xb227,         permanent,
+blake2b-320,                    multihash,      0xb228,         permanent,
+blake2b-328,                    multihash,      0xb229,         permanent,
+blake2b-336,                    multihash,      0xb22a,         permanent,
+blake2b-344,                    multihash,      0xb22b,         permanent,
+blake2b-352,                    multihash,      0xb22c,         permanent,
+blake2b-360,                    multihash,      0xb22d,         permanent,
+blake2b-368,                    multihash,      0xb22e,         permanent,
+blake2b-376,                    multihash,      0xb22f,         permanent,
+blake2b-384,                    multihash,      0xb230,         permanent,
+blake2b-392,                    multihash,      0xb231,         permanent,
+blake2b-400,                    multihash,      0xb232,         permanent,
+blake2b-408,                    multihash,      0xb233,         permanent,
+blake2b-416,                    multihash,      0xb234,         permanent,
+blake2b-424,                    multihash,      0xb235,         permanent,
+blake2b-432,                    multihash,      0xb236,         permanent,
+blake2b-440,                    multihash,      0xb237,         permanent,
+blake2b-448,                    multihash,      0xb238,         permanent,
+blake2b-456,                    multihash,      0xb239,         permanent,
+blake2b-464,                    multihash,      0xb23a,         permanent,
+blake2b-472,                    multihash,      0xb23b,         permanent,
+blake2b-480,                    multihash,      0xb23c,         permanent,
+blake2b-488,                    multihash,      0xb23d,         permanent,
+blake2b-496,                    multihash,      0xb23e,         permanent,
+blake2b-504,                    multihash,      0xb23f,         permanent,
+blake2b-512,                    multihash,      0xb240,         permanent,
+blake2s-8,                      multihash,      0xb241,         draft,     Blake2s consists of 32 output lengths that give different hashes
```

Comment:
Is intentionally Blake2**b** permanent and Blake2**s** draft?

Reply:
I'm not aware of anyone using the blake2s codec in practice.

Reply:
you could also make this argument for most of the blake2b hashes, `blake2b-256` is the only one in _wide_ use (because of Filecoin)